cmd/odf/restore/restore.go

```diff
@@ -0,0 +1,16 @@
+package restore
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// restoreCrd represents the mons command
+var RestoreCrd = &cobra.Command{
+	Use:                "restore",
+	Short:              "Calls subcommands for 'restore'",
```

Comment:
I wonder if we don't need a short description for the higher level commands like this? It just seems obvious already that subcommands will be implemented.

Reply:
I'll remove in separate PR for other commands